build.gradle

```
@@ -4,8 +4,7 @@ plugins {
   id 'groovy'
 }
 
-sourceCompatibility = '1.8'
-targetCompatibility = '1.8'
+release = '11'z
```

Is `z` a typo?
And release is not known either:

```
> Could not set unknown property 'release' for root project 'gatling-gradle-plugin' of type org.gradle.api.Project.
```
(With gradle 7 or gradle 8.5)

Suggestion from stackoverflow: 
```suggestion
tasks.withType(JavaCompile) {
    options.release = 11
}
```

---

Way better:
```suggestion
java {
  toolchain {
    languageVersion.set(JavaLanguageVersion.of(11))
  }
}
```

Because it applies on other Java task (javadoc and test)

---

Thanks. I hate Gradle and its doc.

---

I can relate ^^